addons/website/views/snippets/s_timeline_list.xml

```diff
@@ -0,0 +1,101 @@
+                        <span class="s_timeline_list_dot o_not_editable position-absolute translate-middle-x rounded-circle pe-none text-o-color-1" contenteditable="false"/>
+                        <small class="text-muted">Oct 09, 2024</small>
+                        <b>Mobile App Compatibility Expanded</b>
+                    </div>
+                    <div class="s_timeline_list_row position-relative pb-4" data-name="Milestone">
+                        <div class="s_timeline_list_line position-absolute top-0 bottom-0 w-0 mb-1 border-start pe-none"/>
+                        <span class="s_timeline_list_dot o_not_editable position-absolute translate-middle-x rounded-circle pe-none text-o-color-1" contenteditable="false"/>
+                        <small class="text-muted">Dec 22, 2024</small>
+                        <b>Advanced Analytics Tools Introduced</b>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+</template>
+
+<template id="s_timeline_list_options" inherit_id="website.snippet_options">
+    <xpath expr="//t[@t-call='website.snippet_options_background_options']" position="before">
+        <div data-js="MultipleItems" data-selector=".s_timeline_list">
+            <we-row string="Date">
+                <we-button data-add-item="" data-item=".s_timeline_list_row:first" data-select-item="" data-add-before="true" data-no-preview="true" class="o_we_bg_brand_primary">
+                    Add Date
+                </we-button>
+            </we-row>
+        </div>
+    </xpath>
+    <xpath expr="." position="inside">
+        <div data-selector=".s_timeline_list_row" data-drop-near=".s_timeline_list_row"/>
+        <div data-selector=".s_timeline_list">
+            <we-button-group string="Alignment" title="Alignment" data-apply-to=".s_timeline_list_wrapper">
+                <we-button title="Align Left"
+                        data-select-class="justify-content-start"
+                        class="fa fa-align-left"/>
+                <we-button title="Align Center"
+                        data-select-class="justify-content-center"
+                        class="fa fa-align-center"/>
+                <we-button title="Align Right"
+                        data-select-class="justify-content-end"
+                        class="fa fa-align-right"/>
+            </we-button-group>
+        </div>
+        <div data-selector=".s_timeline_list">
+            <we-colorpicker string="Line Color" data-select-style="true" data-css-property="border-color" data-color-prefix="border-" data-apply-to=".s_timeline_list_line"/>
+        </div>
+        <div data-selector=".s_timeline_list .s_timeline_list_row">
+            <we-colorpicker string="Dot Color" data-select-style="true" data-css-property="color" data-color-prefix="text-" data-apply-to=".s_timeline_list_dot"/>
+        </div>
```

Comment:
Same remark as for the CSS, duplicated options ("Dot Color", etc.) should be be shared between s_timeline and s_timeline_list.